tests/ha/test_control_pod_restart.py

```diff
@@ -0,0 +1,417 @@
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+"""
+HA test suite for Control Pod Restart
+"""
+
+import logging
+import os
+import random
+import secrets
+import threading
+import time
+from multiprocessing import Queue
+from time import perf_counter_ns
+
+import pytest
+
+from commons import commands as cmd
+from commons import constants as const
+from commons.ct_fail_on import CTFailOn
+from commons.errorcodes import error_handler
+from commons.helpers.health_helper import Health
+from commons.helpers.pods_helper import LogicalNode
+from commons.params import TEST_DATA_FOLDER
+from commons.utils import assert_utils
+from commons.utils import system_utils as sysutils
+from config import CMN_CFG
+from config import HA_CFG
+from config.s3 import S3_CFG
+from libs.di.di_mgmt_ops import ManagementOPs
+from libs.ha.ha_common_libs_k8s import HAK8s
+from libs.motr.motr_core_k8s_lib import MotrCoreK8s
+from libs.prov.prov_k8s_cortx_deploy import ProvDeployK8sCortxLib
+from libs.s3.s3_blackbox_test_lib import JCloudClient
+from libs.s3.s3_multipart_test_lib import S3MultipartTestLib
+from libs.s3.s3_rest_cli_interface_lib import S3AccountOperations
+from libs.s3.s3_test_lib import S3TestLib
+from libs.csm.rest.csm_rest_iamuser import RestIamUser
+from http import HTTPStatus
+
+# Global Constants
+LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=R0902
+# pylint: disable=R0904
+class TestControlPodRestart:
+    """
+    Test suite for Control Pod Restart
+    """
+
+    @classmethod
+    def setup_class(cls):
+        """
+        Setup operations for the test file.
+        """
+        LOGGER.info("STARTED: Setup Module operations.")
+        cls.csm_user = CMN_CFG["csm"]["csm_admin_user"]["username"]
+        cls.csm_passwd = CMN_CFG["csm"]["csm_admin_user"]["password"]
+        cls.num_nodes = len(CMN_CFG["nodes"])
+        cls.username = []
+        cls.password = []
+        cls.host_master_list = []
+        cls.node_master_list = []
+        cls.hlth_master_list = []
+        cls.host_worker_list = []
+        cls.node_worker_list = []
+        cls.ha_obj = HAK8s()
+        cls.deploy_lc_obj = ProvDeployK8sCortxLib()
+        cls.s3_clean = cls.test_prefix = cls.random_time = None
+        cls.s3acc_name = cls.s3acc_email = cls.bucket_name = cls.object_name = None
+        cls.restore_pod = cls.deployment_backup = cls.deployment_name = cls.restore_method = None
+        cls.restore_node = cls.node_name = cls.deploy = None
+        cls.restore_ip = cls.node_iface = cls.new_worker_obj = cls.node_ip = None
+        cls.mgnt_ops = ManagementOPs()
+        cls.system_random = secrets.SystemRandom()
+        cls.motr_obj = MotrCoreK8s()
```

Comment:
This won't be needed. Please check [https://github.com/Seagate/cortx-test/pull/1486](url)

Reply:
yeah plz remove this as this will break our test and anyways do u have any RC test in this suite?